src/GRA.Data/Repository/NewsCategoryRepository.cs

```diff
@@ -45,6 +46,15 @@ public async Task<ICollection<NewsCategory>> GetAllAsync(int siteId)
                 .ProjectTo<NewsCategory>(_mapper.ConfigurationProvider)
                 .ToListAsync();
         }
+        public async Task SetLastPostDate(int categoryId, DateTime? date)
+        {
+            var category =  await DbSet
+            .AsNoTracking()
```

Comment:
If you're going to do an immediate update on this record, you shouldn't use `.AsNoTracking()` on it.

Reply:
Also, indent these continuation lines one tab so it's easy to see they are part of the line that starts with `var category =`